aries-site/src/pages/foundation/color.js

```diff
@@ -54,7 +54,8 @@ const page = getPageDetails(title);
 const topic = 'Foundation';
 
 const Color = () => (
-  <Layout title={title}>
+  // Temp isLanding for proper layout until converted to MDX
```

Comment:
Is this represented by an issue in the backlog?

Reply:
It is not, I added it because I was having an issue with converting it and didn't want to hold up the rest of the PR. I will add a backlog issue.

Reply:
It is tracked in #954